lib/active_record_florder.rb

```diff
@@ -55,6 +55,14 @@ def step(value)
     def get_step
       @step || 2**16
     end
+
+    def populate(value)
+      @populate = value
+    end
+
+    def get_populate
```

Comment:
Why not in ruby style like getter/setter (eg. attr_accessible does - I know you have there default) do setter: `def populate(value)` and getter: `def populate`.